packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -59,94 +106,138 @@ const FlipbookControls = ({
-                  <Button
-                    key={`${url}-${index}`}
-                    aria-controls='flipbook-tab-panel'
-                    aria-label={`${t(
-                      'SubjectViewer.MultiFrameViewer.FrameCarousel.thumbnailAltText'
-                    )} ${index + 1}`}
-                    aria-selected={activeFrame ? 'true' : 'false'}
-                    onClick={() => onFrameChange(index)}
-                    onKeyDown={handleKeyDown}
-                    role='tab'
-                    tabIndex={tabIndex}
-                    style={{
-                      display: 'flex',
-                      height: '40px',
-                      width: '40px',
-                      padding: '0',
-                      backgroundImage: `url(${thumbnailerUrl})`,
-                      backgroundSize: 'cover',
-                      backgroundRepeat: 'no-repeat',
-                      backgroundPosition: 'center',
-                      border: activeFrame
-                        ? `solid 2px ${theme.global.colors['neutral-2']}`
-                        : 'none'
-                    }}
-                  />
-                )
-              })}
+    <ThemeContext.Extend value={controlsTheme}>
+      <Box background={backgrounds}>
+        <Grid
+          columns={['120px', 'flex']}
+          pad={{ horizontal: '20px', vertical: '10px' }}
+          gap='small'
+        >
+          {/** Play/Pause & Speed */}
+          <Box direction='row'>
+            <Button
+              a11yTitle={t(playPauseLabel)}
+              onClick={onPlayPause}
+              icon={playing ? <Pause /> : <CirclePlay />}
+              plain
+            />
+            <Select
+              a11yTitle={t('SubjectViewer.VideoController.playbackSpeed')}
+              options={['0.25x', '0.5x', '1x', '2x', '4x']}
+              value={`${playbackSpeed}x`}
+              onChange={handlePlaybackSpeed}
+              plain
+              icon={<FormDown />}
+              focusIndicator
+              width='70px'
+              style={{
+                textAlign: 'right'
+              }}
+            />
+          </Box>
+
+          {/** Image Thumbnails */}
+          <Box direction='row' justify='center' gap='small'>
+            <DirectionButton
+              disabled={currentFrame === 0 || playing}
+              icon={<FormPrevious />}
+              label={t(
+                'SubjectViewer.MultiFrameViewer.FrameCarousel.previousFrameLabel'
+              )}
+              onClick={handlePrevious}
+            />
+            <Box
+              aria-label='Image thumbnails'
+              direction='row'
+              gap='5px'
+              pad='5px'
+              role='tablist'
+            >
+              {locations?.length &&
+                locations.map((location, index) => {
+                  const [url] = Object.values(location)
+                  const thumbnailerUrl = `https://thumbnails.zooniverse.org/100x100${url.slice(
```

Comment:
Not needed for this PR, but the thumbnail service will fail for URLs with origins that aren't owned by Zooniverse. We should add a check that the origin is `https://panoptes-uploads.zooniverse.org`, and fall back to using the full size image for projects that host images on their own domains.

Reply:
Good to note! I added this to the Flipbook Project Board